code/modules/clothing/clothing_accessories.dm

```diff
@@ -1,11 +1,22 @@
-/obj/item/clothing/proc/can_attach_accessory(obj/item/clothing/accessory/A)
-	if(valid_accessory_slots && istype(A) && (A.slot in valid_accessory_slots))
-		.=1
+/obj/item/clothing
+	var/accessory_slot
+	var/accessory_removable = FALSE
+	/// if it should appear on examine without detailed view
+	var/accessory_high_visibility
+	/// used when an accessory is meant to slow the wearer down when attached to clothing
+	var/accessory_slowdown
```

Comment:
I wonder if it's worth having it as a separate variable now that accesory stuff is on the clothing